packages/nimble-components/src/anchor-tabs/index.ts

```diff
@@ -70,14 +76,41 @@ export class AnchorTabs extends FoundationElement {
      */
     public tablist!: HTMLElement;
 
+    /**
+     * @internal
+     */
+    public readonly leftScrollButton!: Element;
+
+    /**
+     * @internal
+     */
+    public readonly tabSlotName = 'anchortab';
+
+    private readonly tabListResizeObserver: ResizeObserver;
     private tabIds: string[] = [];
 
+    public constructor() {
+        super();
+        this.tabListResizeObserver = new ResizeObserver(entries => {
+            let tabListVisibleWidth = entries[0]?.contentRect.width;
+            if (tabListVisibleWidth !== undefined) {
+                const buttonWidth = this.leftScrollButton?.clientWidth ?? 0;
+                tabListVisibleWidth = Math.ceil(tabListVisibleWidth);
+                if (this.showScrollButtons) {
+                    tabListVisibleWidth += buttonWidth * 2;
+                }
+                this.showScrollButtons = tabListVisibleWidth < this.tablist.scrollWidth;
+            }
+        });
+    }
+
     /**
      * @internal
      */
     public activeidChanged(_oldValue: string, _newValue: string): void {
         if (this.$fastController.isConnected) {
             this.setTabs();
+            this.activetab?.scrollIntoView({ block: 'nearest' });
```

Comment:
If you have a long tab name, scrolling into view from the right (i.e. clicking the left arrow button) feels odd because only the end of the tab name becomes visible. I _think_ if you used `inline: 'start'` here, you might get more intuitive behavior.